docs/src/main/asciidoc/authentication.adoc

```diff
@@ -42,6 +42,14 @@ A common challenge for developers is the management of secrets and credentials u
 
 We encourage using manged identity instead of using connection string or key in your application for it's more secure and will save the trouble of managing secrets and credentials. In this case, `DefaultAzureCredential` could better serve the scenario of developing locally using account information stored locally and deploying the application to Azure Cloud and using Manged Identity.
 
+==== Managed Identity Types
+There are two types of managed identities:
+
+- System-assigned Some Azure services allow you to enable a managed identity directly on a service instance. When you enable a system-assigned managed identity an identity is created in Azure AD that is tied to the lifecycle of that service instance. So when the resource is deleted, Azure automatically deletes the identity for you. By design, only that Azure resource can use this identity to request tokens from Azure AD.
+- User-assigned You may also create a managed identity as a standalone Azure resource. You can create a user-assigned managed identity and assign it to one or more instances of an Azure service. In the case of user-assigned managed identities, the identity is managed separately from the resources that use it.
+
+NOTE: When a user-assigned managed identity is used, you can specify it by `spring.cloud.azure.credential.managed-identity-client-id` or `spring.cloud.azure.<azure-service>.credential.managed-identity-client-id`.
```

Comment:
Do we need this: `When system-assigned managed identity is used, ...`

Reply:
yes, if users is looking for configuration properties related to MI, this will help them.

Reply:
But no extra configuration is needed when system-assigned mi is used.

Reply:
Could you please add content about that? Let reader known that.